### PR TITLE
Clean up Section/Course pages

### PR DESCRIPTION
**Pull request by kmcnellis**
Make banner a directive, cleaned up code and added some extra info

### CODE REVIEW

File: client/app/app.less [hunk 1]
```diff
@@ -71,6 +71,54 @@
...
+  .course-banner{
+    display: block;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+    opacity: 0.15;
+    background-image: url('http://placekitten.com/1500/1000');
```
Comment (seveibar) [inline client/app/app.less:110]: ??? placekitten?

**Comment by seveibar** [comment]
@kmcnellis any functional/aesthetic changes here?

**Comment by seveibar** [comment]
Ready to merge after placekitten removal (unless you think it belongs- im worried about it accidentally getting through to production)

**Comment by kmcnellis** [comment]
There are some small visual fixes (alignment)

**Comment by kmcnellis** [comment]
Placekitten was already in the code, I just moved it to a new location. We should choose a new default image later, but I think that should be a separate pr

File: client/app/courses/view/courseview.html [hunk 2]
```diff
@@ -2,15 +2,8 @@
...
-    <div class='row'>
-      <div class="col-md-12 title">
-        <div class="background-image" course-banner course="course"></div>
-          <div class="inside ">
-            <h2> {{ course.name }}</h2>
-            <div class='course-number'>{{course.department}} {{course.courseNumber}}</div>
-          </div>
-        </div>
-      </div>
+      <course-title-banner-basic course="course"/>
```
Comment (jowlee) [inline client/app/courses/view/courseview.html:5]: This course-title-banner-basic directive does not exist @kmcnellis

**Comment by kmcnellis** [comment]
You didn't catch this on PR, seve? Fixed in #266

**Pull request merged**